iris_grib/_save_rules.py

```diff
@@ -665,6 +665,16 @@ def set_fixed_surfaces(cube, grib):
         output_unit = cf_units.Unit('K')
         v_coord = cube.coord("air_potential_temperature")
 
+    elif cube.coords("level_height"):
```

Comment:
level_height is not a CF standard name, is this an Irisism?

I'm not sure about binding 118 to this name